src/main/perl/Service.pm

```diff
@@ -11,9 +11,10 @@ use CAF::Process;
 use LC::Exception qw (SUCCESS);
 
 our $AUTOLOAD;
-use base qw(CAF::Object);
+use base qw(CAF::Object Exporter);
 
 use constant DEFAULT_SLEEP => 5;
+use constant FLAVOURS => qw(linux_sysv linux_systemd solaris);
```

Comment:
Since this is being exported can we switch to ReadOnly please? Otherwise changing this later means changing all the downstream dependencies.